.github/workflows/push-tests.yml

```diff
@@ -217,6 +217,9 @@ jobs:
           - dockerfile: src/backend/Dockerfile.pangolin
             context: ./src/backend/
             name: genepi-pangolin
+          - dockerfile: src/backend/Dockerfile.lineage_qc
+            context: ./src/backend/
+            name: genepi-lineage-qc
```

Comment:
Note - this means we'll need to add the genepi-lineage-qc ECR repository to genepi-infra *before* this pr gets merged.

Reply:
i think we do have that ecr repository created already [here](https://github.com/chanzuckerberg/genepi-infra/blob/9affbe5cd2da4db089a5527f340d1dfb18eca67e/terraform/modules/genepi/main.tf#L32)